dotnetv3/ECS/ECSScenario/ECSScenario.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+using System.Diagnostics;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Amazon.ECS;
+using ECSActions;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Console;
+using Microsoft.Extensions.Logging.Debug;
+
+namespace ECSScenario;
+
+public class ECSScenario
+{
+    /*
+    Before running this .NET code example, set up your development environment, including your credentials.
+
+
+    This .NET example performs the following tasks:
+        1. List ECS Cluster ARNs.
+        2. List services in every cluster
+        3. List Task ARNs in every cluster.
+    */
+
+    private static ILogger logger = null!;
+    private static ECSWrapper _ecsWrapper = null!;
+
+    static async Task Main(string[] args)
+    {
+        // Set up dependency injection for the Amazon service.
+        using var host = Host.CreateDefaultBuilder(args)
+            .ConfigureLogging(logging =>
+                logging.AddFilter("System", LogLevel.Debug)
+                    .AddFilter<DebugLoggerProvider>("Microsoft", LogLevel.Information)
+                    .AddFilter<ConsoleLoggerProvider>("Microsoft", LogLevel.Trace))
+        .Build();
+
+        ILoggerFactory loggerFactory = LoggerFactory.Create(builder =>
+        {
+            builder.AddConsole();
+        });
+
+
+        logger = LoggerFactory.Create(builder => { builder.AddConsole(); })
+            .CreateLogger<ECSScenario>();
+
+
+        var loggerECSWarper = LoggerFactory.Create(builder => { builder.AddConsole(); })
+          .CreateLogger<ECSWrapper>();
+
+
+        var amazonECSClient = new AmazonECSClient();
+
+        _ecsWrapper = new ECSWrapper(amazonECSClient, loggerECSWarper);
+
+        Console.WriteLine(new string('-', 80));
+        Console.WriteLine("Welcome to the Amazon ECS example scenario.");
+        Console.WriteLine(new string('-', 80));
+
+        try
+        {
+            await ListClusterARNs();
+            await ListServiceARNs();
+            await ListTaskARNs();
+
+        }
+        catch (Exception ex)
+        {
+            logger.LogError(ex, "There was a problem executing the scenario.");
+            //await CleanupResources();
```

Comment:
You can remove this if there are no resources created that need to be cleaned up.